docs/changelog.md

```diff
@@ -4,6 +4,12 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v2.3.0 - Apr 27, 2022
+### Bug Fixes
```

Comment:
```suggestion
## v2.3.0 - Apr 27, 2022

### Bug Fixes
```

Reply:
In general I think `Bug Fixes` should go after `Major Features and Improvements`.